src/app/strategies/[strategyId]/page.tsx

```diff
@@ -11,6 +11,7 @@ import Pagination from '../../../component/pagination';
 import '../../globals.scss';
```

Comment:
If possible, try to place it under the pages.

Reply:
> place it under the pages.

Due to the dynamic routing problem, we're still tring to figure out how to do it. Will do in another branch.

Reply:
> If possible, try to place it under the pages.

I'll do this in another branch.